android/app/build.gradle

```diff
@@ -33,7 +33,7 @@ android {
 
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "com.thealphamerc.flutter_twitter_clone_dev"
```

Comment:
@aa-rodriguezv application Id cannot be changed, so please revert this to old one.